src/controllers/controlpickermenu.cpp

```diff
@@ -628,6 +628,15 @@ ControlPickerMenu::ControlPickerMenu(QWidget* pParent)
     QString beatJumpBackwardDescription = tr("Jump backward by %1 beats, or if a loop is enabled, move the loop backward %1 beats");
     addDeckControl("beatjump_forward", tr("Beat Jump / Loop Move Forward Selected Beats"), tr("Jump forward by the selected number of beats, or if a loop is enabled, move the loop forward by the selected number of beats"), beatJumpMenu);
     addDeckControl("beatjump_backward", tr("Beat Jump / Loop Move Backward Selected Beats"), tr("Jump backward by the selected number of beats, or if a loop is enabled, move the loop backward by the selected number of beats"), beatJumpMenu);
+    addDeckControl("beatjump_anchor",
```

Comment:
How about calling it just "loop_anchor"? A beat jump can be done without a loop and will jump the entire loop without in and out"